gsplat/_torch_impl.py

```diff
@@ -133,11 +133,16 @@ def quat_to_rotmat(quat: Tensor) -> Tensor:
     return mat.reshape(quat.shape[:-1] + (3, 3))
 
 
+def quat_to_rotmat(quat: Tensor) -> Tensor:
+    assert quat.shape[-1] == 4, quat.shape
+    return normalized_quat_to_rotmat(F.normalize(quat, dim=-1))
+
+
 def scale_rot_to_cov3d(scale: Tensor, glob_scale: float, quat: Tensor) -> Tensor:
     assert scale.shape[-1] == 3, scale.shape
     assert quat.shape[-1] == 4, quat.shape
     assert scale.shape[:-1] == quat.shape[:-1], (scale.shape, quat.shape)
-    R = quat_to_rotmat(quat)  # (..., 3, 3)
+    R = normalized_quat_to_rotmat(quat)  # (..., 3, 3)
```

Comment:
The cuda code currently doesn't implement the backprop gradient of normalize(...).